README.md

```diff
@@ -4,8 +4,153 @@ Measurement library will provide an open source alternative for sites/apps to se
 
 **This is not an officially supported Google product.**
 
-# Running tests and creating the build
 
+[![Build Status](https://travis-ci.org/googleinterns/measurement-library.svg?branch=master)](https://travis-ci.org/github/googleinterns/measurement-library/branches)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange.svg)
+
+## Table of contents
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Event Command](#event-command)
+    - [Set Command](#set-command)
+- [Event Processors](#event-processors)
+    - [Google Analytics](#google-analytics)
+    - [Custom Event Processor](#custom-event-processor)
+- [Storage Interfaces](#storage-interfaces)
+    - [Cookies](#cookies)
+    - [Custom Storage Interface](#custom-storage-interface)
+- [Local Setup](#local-setup)
+    - [Creating the build](#creating-the-build)
+    - [Running Tests Interactively](#running-tests-interactively)
+
+# Overview
+Measurement Library is a utility to help developers send data collected from their website to
+other libraries for analytics. After specifying settings in a script tag, you can process any
+sent events using a variety of built in event processors (e.g. sending data to Google Analytics).
+
+Data about the user is automatically saved to persist between visits using your choice of
+a variety of different storage options (e.g. cookies).
+
+# Installation
+First, install the package with your favorite package manager:
+```bash
+yarn add measurement-library
+// or
+npm install measurement-library
+```
+
+Next, choose [the event processor](#event-processors) and [the storage implementation](#storage-interfaces)
+that you want to use and include this code in your `<head>` tag.
```

Comment:
Our current docs for Google scripts say "immediately after the <head> tag" or "as high up in the <head> block as possible" which I like since it highlights that it should be loaded as soon as possible.  We recommend this to catch as many hits for "bounced" users as possible.  Basically, users who load the page then immediately leave.

Our current gtag docs don't mention the reason, though I think older docs did.

Reply:
Done. I added the reason because more context is better